services/queue/config.yml

```diff
@@ -109,7 +109,9 @@ defaults:
     secretAccessKey:  !env AWS_SECRET_ACCESS_KEY
     region:           !env ARTIFACT_REGION
     endpoint:         !env:optional AWS_ENDPOINT
+    s3ForcePathStyle: !env:bool:optional AWS_FORCE_PATH_STYLE
```

Comment:
@Eijebong I don't see this config value is being used anywhere. Is it because aws libs are reading this env var `AWS_FORCE_PATH_STYLE` and change their behavior?

Reply:
got it! Thanks